blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/NumericSerializer.java

```diff
@@ -332,14 +333,38 @@ private static SetRollup deserializeV1SetRollup(CodedInputStream in) throws IOEx
         }
         return rollup;
     }
-    
-    private static void serializeTimer(TimerRollup rollup, byte[] buf) throws IOException {
+
+    private static void serializeV1Timer(TimerRollup rollup, byte[] buf) throws IOException {
         CodedOutputStream out = CodedOutputStream.newInstance(buf);
         timerRollupSize.update(buf.length);
         out.writeRawByte(Constants.VERSION_1_TIMER);
+
+        // sum, count, countps, avg, max, min, var
+        out.writeRawVarint64((long)rollup.getSum());
```

Comment:
rollup.getSum() has been converted into a double. So to serialize it in the old way we need to cast it to long